### DISCUSSION TITLE
Improve PowerShell markdown style

### DISCUSSION THREAD

**Discussion by SpitFire-666** [community/community#57287] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It would be nice if PowerShell syntax highlighting was improved. This would make it easier to read

Some callouts:
- Highlighting of variables ($vars) is inconsistent
- There's lots of blue. Variables, integers, commands and even stuff in quotes. Some variation in colours would be nice! 

For dark mode github, you could replicate the colour scheme of Terminal or PowerShell - eg dark green comments, bright green variables, yellow commands etc.

Some pics showing what I mean:

![image](https://github.com/community/community/assets/38451588/32642ad8-2718-45c5-8692-4fe56fd82ec4)

![image](https://github.com/community/community/assets/38451588/d27930fd-8213-4174-a67f-83838c9c6b18)

![image](https://github.com/community/community/assets/38451588/d38ad4c1-d106-4360-8de6-f1b69d996f2d)

![image](https://github.com/community/community/assets/38451588/7eb62236-0ab5-460c-9a35-985a9c567126)

![image](https://github.com/community/community/assets/38451588/e6ae3b28-f0f2-4c5e-9b32-8df9703c635f)

Cheers!

**Comment by zm-cttae** [comment]
You could suggest this at the Powershell grammar and see what happens.

https://github.com/PowerShell/EditorSyntax/issues
